.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Auto publish to NPM
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 21
+      - run: npm install
+      - run: npm run build
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 21
+          registry-url: https://registry.npmjs.org/
+      - run: npm publish
```

Comment:
Just to make sure. Should we use npm or bun here? Or we will start with npm and update to bun in the future?

Reply:
I always use npm to publish, I'm not sure it's works with bun.

But installing packages maybe we can do with bun, if is supported and straight forward. Otherwise I think it's best to keep`npm` for simplicity.